tensorflow_probability/python/bijectors/real_nvp.py

```diff
@@ -20,7 +20,7 @@
 
 import tensorflow as tf
 from tensorflow.python.layers import core as layers
-from tensorflow.python.ops.distributions import bijector
+from tensorflow_probability.python.bijectors.conditional_bijector import ConditionalBijector
```

Comment:
Prefer to import the module vs. the class:

from tensorflow_probability.python.bijectors import conditional_bijector

Reply:
Thanks!